drivers/bridge/bridge_test.go

```diff
@@ -72,6 +72,71 @@ func TestCreateFail(t *testing.T) {
 	}
 }
 
+func TestCreateMultipleNetworks(t *testing.T) {
+	defer netutils.SetupTestNetNS(t)()
+	d := newDriver()
+	dd, _ := d.(*driver)
+
+	config1 := &networkConfiguration{BridgeName: "net_test_1", AllowNonDefaultBridge: true}
```

Comment:
Since we are testing the iptables rules, shouldnt you enable iptables explicitely for each of the networks ?
Atleast until before https://github.com/docker/libnetwork/pull/211 is merged :)